checkers/python/avoid-allowing-all-hosts.tests.py

```diff
@@ -0,0 +1,15 @@
+# Match 
+# <expect-error>
+ALLOWED_HOSTS = ["*"]
```

Comment:
Can we detect if `ALLOWED_HOSTS` is overwritten further in the file? And if we can check for conditionals?